src/main/java/org/mondemand/transport/LWESTransport.java

```diff
@@ -312,8 +312,7 @@ protected int updateLwesEventForSamples(Event event, SamplesMessage msg,
         }
       }
     }
-    // clear the samples in the sampleMessage
-    msg.resetSamples();
```

Comment:
the snapshot should be GC-ed after dispatch, so there is no need to reset the data here or elsewhere.